noxfile.py

```diff
@@ -191,6 +191,7 @@ def test_emscripten(session: nox.Session):
     )
     session.env["CARGO_BUILD_TARGET"] = target
     session.env["PYO3_CROSS_LIB_DIR"] = pythonlibdir
+    session.env["PYTHONDONTWRITEBYTECODE"] = "1"
```

Comment:
I'm pretty sure this won't work because it doesn't set the environment variable inside the VM.